Makefile

```diff
@@ -614,6 +614,7 @@ test-e2e:
 	E2E=ok K8S_VERSION=$(K8S_VERSION) go test ./test/e2e/metrics -v
 	E2E=ok K8S_VERSION=$(K8S_VERSION) go test ./test/e2e/mutate -v
 	E2E=ok K8S_VERSION=$(K8S_VERSION) go test ./test/e2e/generate -v
+	E2E=ok K8S_VERSION=$(K8S_VERSION) go test -timeout 30m ./test/e2e/validate -v
```

Comment:
We decided to migrate e2e test to kuttl tests:

https://kubernetes.slack.com/archives/C032MM2CH7X/p1669558437288979

Can this test be converted to a kuttl test?